### PR TITLE
Ceph-ci integradtion of CEPH-83574626

### PR DESCRIPTION
**Pull request by ckulal**
# Description

Ceph-ci integration of polarian: CEPH-83574626
MS-DBR - Disable resharding feature on a greenfield cluster

**dependent on ceph-qe pr: https://github.com/red-hat-storage/ceph-qe-scripts/pull/469**

Log:
6.1: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-pog6e/
5.3:  http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-estgw/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by anrao19 — APPROVED** [review]

**Review by viduship — DISMISSED** [review]

File: pipeline/metadata/6.0.yaml [hunk 1]
```diff
@@ -901,6 +901,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "test-rgw-ms-dbr-disable"
+          execution_time: "1h 37m 31s"
+          suite: "suites/quincy/rgw/tier-2-_rgw_ms-dbr-disable.yaml"
```
Comment (mkasturi18) [inline pipeline/metadata/6.0.yaml:906]: @ckulal , could you fix the typo in the suite file name

File: suites/quincy/rgw/tier-2_rgw_ms-dbr-disable.yaml
Comment (mkasturi18) [inline suites/quincy/rgw/tier-2_rgw_ms-dbr-disable.yaml:1]: This file is not being called in the pipeline

File: pipeline/metadata/6.0.yaml (hunk 1 above)
Comment (ckulal) [reply pipeline/metadata/6.0.yaml]: done

File: suites/quincy/rgw/tier-2_rgw_ms-dbr-disable.yaml
Comment (ckulal) [reply suites/quincy/rgw/tier-2_rgw_ms-dbr-disable.yaml:1]: This suite is used in 6.0 and 6.1 metadata file,
but there was extra '-' was added in metafile have removed now.

**Pull request merged**